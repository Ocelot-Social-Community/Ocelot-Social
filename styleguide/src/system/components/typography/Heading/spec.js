import { shallowMount } from '@vue/test-utils'
import Comp from './Heading.vue'

describe('Heading.vue', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Comp, {
      slots: {
        default: 'Winter is coming'
      }
    })
  })

  it('defaults to h1', () => {
    expect(wrapper.props().tag).toEqual('h1')
  })

  it('displays title', () => {
    expect(wrapper.text()).toEqual('Winter is coming')
  })

  it('matches snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})