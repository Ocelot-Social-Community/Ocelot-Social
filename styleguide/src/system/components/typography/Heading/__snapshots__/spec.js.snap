// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading.vue matches snapshot 1`] = `
<h1
  class="ds-heading ds-heading-h1"
>
  Winter is coming
</h1>
`;
